src/components/Login.svelte

```diff
@@ -0,0 +1,121 @@
+<script>
+import { login } from '../data/auth'
+import { querystring } from 'svelte-spa-router'
+import qs from 'qs'
+import { me } from '../data/user'
+import jquery from 'jquery' // $ is already a reserved word in svelte
+
+export let placeholder = 'Enter email address'
+export let buttonText = 'Sign in'
+export let hideRememberMe = false
+export let returnTo = qs.parse($querystring)['return-to']
+
+let email = getStoredEmail() || ''
+let checked = !!email
+let identityProviders = []
+
+function getStoredEmail() {
+  return localStorage.getItem('email')
+}
+
+function storeRememberMeChoice() {
+  if (checked) {
+    localStorage.setItem('email', email)
```

Comment:
When/if it's available, switch to using LIFESPAN.LONG

Reply:
good catch:  https://github.com/silinternational/wecarry-ui/pull/139/commits/6e4364f39700df3e3eb723d2547b33a0d559d943